modules/ting_search/plugins/content_types/search_result_count.inc

```diff
@@ -22,25 +22,24 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
 
   $search_result = ting_search_current_results();
   if (isset($search_result)) {
-    $results = $search_result->getNumTotalObjects();
+    // OpenSearch only delivers the number of manifestions
+    // but the search result can also include materials.
+    // Because of this, we need to display different types
+    // of result counts based on the "more results" value,
+    // which basically checks if theres more than one page
+    // available for the search result pagination.
```

Comment:
Is there a reason why the comment is so cramped? Drupals Coding Standards requires a max line length of 80 chars. This seems to break at 60.